WalletWasabi.Fluent/Styles/Themes/Base.axaml

```diff
@@ -64,6 +64,10 @@
     <SolidColorBrush x:Key="PrivacyLevelStrongBrush" Color="#3E992F" />
     <SolidColorBrush x:Key="PrivacyLevelNoneBrush" Color="#B8B8B8" />
 
+    <SolidColorBrush x:Key="PositiveBrush" Color="#5BB626" />
+    <SolidColorBrush x:Key="NegativeBrush" Color="#C3983B" />
+    <SolidColorBrush x:Key="ZeroBrush" Color="#C3983B" />
```

Comment:
ZeroBrush and NegativeBrush shouldn't be the same color right?

Reply:
They can be, because in fact, there's no negative color defined in the specs. Should we define one? @soosr

Reply:
Just have a `PositiveBrush` and a `UncertainBrush`. Later if we need a negative brush, we can add one.